src/intentHandlers/ProjectStatus.js

```diff
@@ -269,6 +269,110 @@ const newlyCreated = async ((kanbanBoardID, StatusIDs, middleTS, oldestTS, subQu
     return newlyCreated;
 });
 
+// Checks if there are any red columns and returns array of objects consisting of the results
+const redColumnCheck = async((kanbanBoardID, boardConfig) => {
+    var redColumns = [];
+    const columns = boardConfig.columnConfig.columns;
+    for (let i = 0; i < columns.length; i++) {
+        var statusIDs = [];
+        for (j = 0; j < columns[i].statuses.length; j++) {
+            statusIDs.push(columns[i].statuses[j].id);
+        }
+        if (columns[i].name.toUpperCase() !== 'BACKLOG' && statusIDs != []) {
+            var jiraResponse = await(JiraService.boardInfo(`${kanbanBoardID}/issue?jql=status in (${statusIDs.join(',')}) and issueType!= Epic and resolution is EMPTY ORDER BY created DESC`)
+                .catch(error => { throw new Error(`error getting redColumnCheck: ${error}`) })); 
+            if (jiraResponse.issues.length > columns[i].max){
+                redColumns.push({
+                    "text": `:exclamation: The ${columns[i].name} column is overflowing with the following issues:${returnIssuesWithAssignee(jiraResponse)}`,
+                    "color": "danger",
+                    "mrkdwn_in": ["text"]
+                });
+            }
+        }
+    }
+    if (redColumns.length == 0){
+        redColumns.push({
```

Comment:
`redColumns` is now misnamed.

I would consider rewriting this to be something like:
```
const allOk = [{"text": "blah", ...}];
return (redColumns.length === 0) ? allOk : redColumns
```

Reply:
It may be an interesting exercise for you to re-write this method using lodash-fp or similar to eliminate the mutable `redColumns` altogether.

Reply:
I will look into it.

Reply:
Done.